package.json

```diff
@@ -181,6 +181,7 @@
     ]
   },
   "dependencies": {
+    "exceljs": "^4.3.0",
```

Comment:
Does it make sense for direct dependencies to be locked with `~`?

Reply:
I don't have an opinion on this subject. I had a look at core repo, all the dependencies are with `^`